api/v1alpha1/workspace_validation_test.go

```diff
@@ -82,59 +125,16 @@ func TestResourceSpecValidateCreate(t *testing.T) {
 				InstanceType: "Standard_NV12s_v3",
 				Count:        pointerToInt(1),
 			},
-			inferenceSpec: &InferenceSpec{
-				Template: &v1.PodTemplateSpec{}, // Assuming a non-nil TemplateSpec implies it's set
-			},
+			preset:     false,
 			errContent: "",
 			expectErrs: false,
 		},
-		{
-			name: "Invalid Preset",
-			resourceSpec: &ResourceSpec{
-				InstanceType: "Standard_NV12s_v3",
-				Count:        pointerToInt(1),
-			},
-			inferenceSpec: &InferenceSpec{
-				Preset: &PresetSpec{
-					PresetMeta: PresetMeta{
-						Name: ModelName("invalid-preset"),
-					},
-				},
-			},
-			errContent: "Unsupported preset",
-			expectErrs: true,
-		},
-
-		{
-			name: "Invalid SKU",
-			resourceSpec: &ResourceSpec{
-				InstanceType: "Standard_invalid_sku",
-				Count:        pointerToInt(1),
-			},
-			inferenceSpec: &InferenceSpec{
-				Preset: &PresetSpec{
-					PresetMeta: PresetMeta{
-						Name: ModelName("llama-2-70b"),
-					},
-				},
-			},
-			errContent: "Unsupported instance",
-			expectErrs: true,
-		},
-
```

Comment:
why remove tests

Reply:
Invalid SKU is duplicated and invalid preset case will not happen since the error will be caught by previous checks.